modules/book.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable quotes */
```

Comment:
- [ ] Its allowed that you can disable only `max-classes-per-file` error 😊 but disable error for `quotes` is not good 🤔 Kindly remove line 2 and fix this error manually 👍
- [ ] Kindly remove all these disabling error lines in all javascript files and fix these errors manually 👍